mantidimaging/core/operations/flat_fielding/flat_fielding.py

```diff
@@ -235,10 +243,13 @@ def register_gui(form, on_change, view) -> dict[str, Any]:
         }
 
     @staticmethod
-    def execute_wrapper(  # type: ignore
-            flat_before_widget: DatasetSelectorWidgetView, flat_after_widget: DatasetSelectorWidgetView,
-            dark_before_widget: DatasetSelectorWidgetView, dark_after_widget: DatasetSelectorWidgetView,
-            selected_flat_fielding_widget: QComboBox, use_dark_widget: QCheckBox) -> partial:
+    def execute_wrapper(**kwargs):
+        flat_before_widget = kwargs['flat_before_widget']
+        flat_after_widget = kwargs['flat_after_widget']
+        dark_before_widget = kwargs['dark_before_widget']
+        dark_after_widget = kwargs['dark_after_widget']
+        selected_flat_fielding_widget = kwargs['selected_flat_fielding_widget']
+        use_dark_widget = kwargs['use_dark_widget']
```

Comment:
These should stay as normal arguments. Going though kwargs makes type checking less useful.